packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -28,11 +28,14 @@ RUN dnf -y install dnf-plugins-core epel-release && \
                    python3-coveralls \
                    python3-requests \
                    python3-docopt \
+                   lldpd \
+                   tcpreplay \
                    && \
     alternatives --set python /usr/bin/python3 && \
     ln -s /usr/bin/pytest-3 /usr/bin/pytest && \
     dnf clean all && \
     systemctl enable openvswitch && \
+    systemctl enable lldpd && \
```

Comment:
Why is this service needed?

Reply:
That service is needed in order to listen on lldp packets

Reply:
It is not needed by NetworkManager, so something else seems to be the problem here.

Reply:
I am investigating this. I have a system where this is working without lldpd, now I am trying to know why is not working on the container or my VM.

Reply:
Indeed, it is working in VM and centos container without lldpd. I might get confused or something because I thought I tried this without lldpd and it was not working. Never mind, thanks for pointing this out :-)